patches/chromium/chore_allow_overriding_of_enable_pak_file_integrity_checks.patch

```diff
@@ -0,0 +1,34 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@electronjs.org>
+Date: Tue, 11 May 2021 21:35:39 -0400
+Subject: chore: allow overriding of enable_pak_file_integrity_checks
+
+enable_pak_file_integrity_checks is not currently overridable
+but Electron does not need to run this.
```

Comment:
is this something we want to actively avoid? Can we make it work instead? I'd rather avoid a patch here if we can.

Reply:
In talking with @nornagon on this, we would have to add code electron to check pak integrity, eg: https://chromium.googlesource.com/chromium/src/+/b49a6bb77903ab52f539014f3482855ac81d9728%5E%21/#F3. We agreed this can be handled as a separate feature PR.